os/arch/arm/src/armv7-m/up_doirq.c

```diff
@@ -104,6 +103,7 @@ uint32_t *up_doirq(int irq, uint32_t *regs)
 #ifdef CONFIG_ARCH_NESTED_INTERRUPT_STACKCHECK
 
 	/* check remaining stack available, assert if stack overflows */
+	uint32_t stack_remain;
```

Comment:
PR description says error message but it looks warning. can you change description

Reply:
And how about titling like `arm/src/armv7-m/up_doirq.c: remove unused variable`? This is better to know what it is I think.

Reply:
Done